CHANGELOG.md

```diff
@@ -7,6 +7,8 @@
 - Depend on newer version of gRPC package in the NodeJS SDK. This version has
   prebuilt binaries for Node 12, which should make installing `@pulumi/pulumi`
   more reliable when running on Node 12.
+- Add `pulumi query` command for querying cloud resources. Includes support for querying state
```

Comment:
Can we just drop this from the changelog for now given that we are hiding the command behind the a debug flag (and then add it back when we actually have the E2E MVP)?

Reply:
Yes we can.